backend/inspirehep/records/api/literature.py

```diff
@@ -175,6 +176,8 @@ def update(self, data, disable_orcid_push=False, *args, **kwargs):
                 push_to_orcid(self)
             self.push_authors_phonetic_blocks_to_redis()
 
+            push_to_hal(self)
```

Comment:
we need a boolean flag so we can disable this in the migrator (like for `disable_orcid_push`). Question @drjova: rename `disable_orcid_push` -> `disable_external_push` and use it for both, or have a separate flag? I would be in favor of the first option to have a cleaner API, but it's up to you. @oguzdemirbasci you'll also need to adapt the migrator to perform the `hal_push` separately if needed in the same way we do the `orcid_push` currently.

Reply:
Totally agree, let's have one flag, we already have enough of them